renderdoc/driver/vulkan/vk_counters.cpp

```diff
@@ -85,6 +195,34 @@ CounterDescription VulkanReplay::DescribeCounter(GPUCounter counterID)
     }
   }
 
+  if(IsKHRCounter(counterID))
+  {
+    const VkPerformanceCounterKHR& khrCounter =
+      m_KHRCounters[FromKHRCounter(counterID)];
+    const VkPerformanceCounterDescriptionKHR& khrCounterDesc =
+      m_KHRCountersDescriptions[FromKHRCounter(counterID)];
+
+    CounterDescription rdcDesc;
+    rdcDesc.counter = counterID;
+    rdcDesc.name = khrCounterDesc.name;
+    rdcDesc.category = khrCounterDesc.category;
+    rdcDesc.description = khrCounterDesc.description;
+
+    auto unitIter = m_KHRCounterUnitsMap.find(khrCounter.unit);
+    if (unitIter != m_KHRCounterUnitsMap.end())
+    {
+        rdcDesc.unit = unitIter->second.unit;
+        rdcDesc.resultType = unitIter->second.type;
+        rdcDesc.resultByteWidth = unitIter->second.byteWidth;
+    }
+    else
+      RDCWARN("Unknown KHR counter unit %i, please fix Renderdoc.", khrCounter.unit);
```

Comment:
Formatting nit: I prefer to have braces on both if and else to match - i.e. if either path needs braces then both should have braces.